opennms-container/horizon/Dockerfile

```diff
@@ -71,7 +71,7 @@ RUN echo "installing packages for build ${REVISION} (${BUILD_DATE})"; \
     if [[ "$(ls -1 /tmp/tarball/*.tar.gz 2>/dev/null | wc -l)" != 0 ]]; then \
         mkdir -p /opt/opennms && \
         tar -xzf /tmp/tarball/opennms-*.tar.gz -C /opt/opennms && \
-        chown -R 0:0 /opt/opennms && \
+        chown -R 10001:10001 /opt/opennms && \
```

Comment:
I remember we have introduced this with the requirement to run in OpenShift which needs support to run with arbitrary user ids. @agalue can you double-check here if this is still the case?